site/current-book/metron-platform/metron-common/index.html

```diff
@@ -577,7 +577,12 @@ <h1>Performance Logging</h1>
 </pre></div></div>
 
 <p><b>Configuration</b></p>
-<p>The first argument to the logger is a java.util.function.Supplier&lt;Map&lt;String, Object&gt;&gt;. The offers flexibility in being able to provide multiple configuration &#x201c;suppliers&#x201d; depending on your individual usage requirements. The example above, taken from org.apache.metron.enrichment.bolt.GenericEnrichmentBolt, leverages the global config to dymanically provide configuration from Zookeeper. Any updates to the global config via Zookeeper are reflected live at runtime. Currently, the PerformanceLogger supports the following options:</p>
+<p>The first argument to the logger is a java.util.function.Supplier&lt;Map&lt;String, Object&gt;&gt;.
```

Comment:
Probably don't want to change the site book.

Reply:
Hm, that's odd. Must have been an IntelliJ mixup.